sites/example-project/src/components/modules/formats.js

```diff
@@ -0,0 +1,126 @@
+import * as ssf from "ssf";
+import { getContext } from "svelte";
+
+//see https://www.benlcollins.com/spreadsheets/google-sheets-custom-number-format/
+//TODO note 0.##% is not ideal as it could show something like 15.% => https://superuser.com/questions/205759/format-a-number-with-optional-decimal-places-in-excel
+export const builtInFormats = [
+  // Date/Time:
+  {
+    formatName: "date",
+    formatValue: "MMM dd, yyyy",
+    valueType: "date",
+    exampleInput: "Jan 9 2022 07:32:04",
+  },
+  {
+    formatName: "month",
+    formatValue: "mmm",
+    valueType: "date",
+    exampleInput: "2022-01-09",
+  },
+  {
+    formatName: "year",
+    formatValue: "yyyy",
+    valueType: "date",
+    exampleInput: new Date("Sun Jan 9 2022 07:32:04 GMT-0400"),
+  },
+
+  // Currency:
+  {
+    formatName: "usd",
+    formatValue: "$#,##0.00;($0.00)",
```

Comment:
1. I'd consider not displaying decimals by default here, so: `$#,##0`
I am very rarely dealing with $ values <$100 in business reporting.
Anything > $100 you definitely dont want the precision

2. Rather than use () for negative, (which is the accounting format), I'd probably use the negative sign, which is more common in business: `-$#,##0`

3. I am unclear on why the negative value is in a different format to the positive value

So the composite string would be (for all currency formats)

`"$#,##0;-$#,##0"`

Reply:
> 1. I'd consider not displaying decimals by default here, so: `$#,##0`
>    I am very rarely dealing with $ values <$100 in business reporting.
>    Anything > $100 you definitely dont want the precision


> 2. Rather than use () for negative, (which is the accounting format), I'd probably use the negative sign, which is more common in business: `-$#,##0`

Fair enough. The inspiration was that this notation is common in finance for negative currency values, but regardless I think I got a bit carried away with the capabilities of this formatting system :D.  I'll take out the special formatting for negative currency values.  i.e `$#,##0` should be sufficient and if anything consistent with what we have right now.  I think this covers 3) as well.